config/settings/tailer.q

```diff
@@ -48,6 +48,7 @@ compression:()
 gc:1b                                                                                       // garbage collect at appropriate points (after each table save and after sorting data)
 eodwaittime:0D00:00:10.000                                                                  // time to wait for async calls to complete at eod
 tpcheckcycles:0W                                                                            // number of attempts to connect to tp before process is killed
+maxrows:10000										    // number of rows of data before savedown occurs during tplog replay
```

Comment:
the comment on this line is aligned using tabs, other lines use spaces

Reply:
Okay will change that. Just for future reference, how can you see if you've used spaces instead of indents or viceversa?

Reply:
I use refined github extension https://github.com/refined-github/refined-github - this shows tabs & spaces to make it obvious. I think there is probably a way to enable something similar in vim or other text editors

Reply:
Thanks will have a look at that.